packages/api/test/unit/builder/oapiSpec.test.ts

```diff
@@ -12,7 +12,7 @@ import {testData} from "./testData.js";
 // Solutions: https://stackoverflow.com/questions/46745014/alternative-for-dirname-in-node-js-when-using-es6-modules
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
-const version = "v0.2.0";
+const version = "v0.5.0";
```

Comment:
Can't believe we were on 0.2.0 which was released in 2022 🤦

Reply:
Not much changed since then, the only difference are new schemas and in the latest release also ssz support